src/common/Utilities.py

```diff
@@ -60,12 +60,13 @@ def examples_data_path(data_type):
     try:
         py_root = sirf.__file__
         try:
-            # is install dir like $SIRF_PATH/lib/python3.8/site-packages?
+            # is install dir like $SIRF_INSTALL_PATH/lib/python?.?/site-packages?
             lib_idx = py_root.split(os.sep).index('lib')
         except ValueError:
-            # is install dir like $SIRF_PATH/python?
+            # is install dir like $SIRF_INSTALL_PATH/python?
             lib_idx = py_root.split(os.sep).index('python')
-        potential_install_loc = os.path.join(os.sep, *py_root.split(os.sep)[:lib_idx])
+        potential_install_loc = os.path.abspath(
+            os.path.join(os.sep, *py_root.split(os.sep)[:lib_idx]))
```

Comment:
Hmm, I'm confused. I don't have SIRF installed on Windows to replicate, but I've tried a minimum example.

On Windows:
```python
>>> import os                                                                                                                     
>>> py_root = os.__file__                                                                                                         
>>> lib_ids = py_root.split(os.sep).index('lib')                                                                                  
>>> lib_idx = py_root.split(os.sep).index('lib')                                                                                  
>>> potential_install_loc = os.path.abspath(os.path.join(os.sep, *py_root.split(os.sep)[:lib_idx]))                               
>>> print(potential_install_loc)                                                                                                  
C:\Windows\system32\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.1520.0_x64__qbz5n2kfra8p0 
```

On Linux:
```python
In [7]: import os                                                                               
In [8]: py_root = os.__file__                                                                   
In [9]: lib_idx = py_root.split(os.sep).index('lib')                                            
In [10]: potential_install_loc = os.path.abspath(os.path.join(os.sep, *py_root.split(os.sep)[:li
    ...: b_idx]))                                                                               
In [11]: print(potential_install_loc)                                                           
/home/gil2a4/anaconda3
```

So I don't get the repetition that you get. Indeed, a quick Google suggests that `os.path.abspath(os.path.join(os.sep, *PATH))` is a [recommended way of getting the absolute version of PATH on Windows and Linux](https://stackoverflow.com/a/28080468)

Reply:
Tried your Windows example, got this:
```
C:\Users\wps46139\Documents\GitHub\build\sources\SIRF\examples\Python\PET\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64
```

Reply:
how about this (on Windows):
```
root_split = sirf.__file__.split(os.sep)
idx = root_split.index('python')
potential_install_loc = root_split[0] + os.path.join(os.sep, root_split[1 : idx])
```

Reply:
@evgueni-ovtchinnikov  - that seemed to work for me. On Windows root_split[0] is the drive, and on Unix it is empty, so this satisfies both. This seems kind-of "cowboy", but I've given this some diligence and I'm surprised to no find much!

Probably there is something more elegant using pathlib, but since that's not used elsewhere this would be more rather than less confusing for other developers?

Anyway, check out the latest commit below